cccm-frontend/src/components/crna-cmp/crnaCmpForm.vue

```diff
@@ -439,6 +432,7 @@ export default {
     },
     handleCancelForm() {
       console.log("Cancel Form");
+      this.$emit('cancelFormClicked');
```

Comment:
I assume this will call the delete endpoint once we have it :-)
We'll also want to check the status of the form first (and probably an additional check on the back end just in case multiple people edit a form in future)

Reply:
Correct.